bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -106,7 +107,10 @@ final class BagsManager(executorService: ExecutorService) {
         case AutomaticGroup.Initial | AutomaticGroup.Active(_,_) => true
         case _                                                   => false
       }
-      enabledGroup && (ctx.getInstrument.getType != SPComponentType.INSTRUMENT_GPI)
+      enabledGroup &&
+        (ctx.getInstrument.getType != SPComponentType.INSTRUMENT_GPI) &&
```

Comment:
That's correct. There's no `AgsStrategy` for GPI, and my understanding is that GPI doesn't rely on a guidestar?

There was a JIRA issue specifically for this, but I can't see to find it now.